stdlib/2and3/asyncore.pyi

```diff
@@ -47,10 +47,15 @@ class dispatcher:
     def set_reuse_addr(self) -> None: ...
     def readable(self) -> bool: ...
     def writable(self) -> bool: ...
+    def listen(self, backlog: int) -> None: ...
+    def bind(self, address: Union[tuple, str]) -> None: ...
+    def connect(self, address: Union[tuple, str]) -> None: ...
     def accept(self) -> Optional[Tuple[socket.socket, Any]]: ...
-    def recv(self, buffer_size: int) -> str: ...
-    def log(self, message: Any) -> None: ...
+    def send(self, data: bytes) -> int: ...
```

Comment:
To reply to the now-concealed comment (sorry about that!): it looks like this was always the correct signature for dispatcher's `send` – `flags` never existed here, only in `socket`, and as far as I can tell the return type was never `Optional`.